### PR TITLE
rename globalState to wizardContext

### PR DESCRIPTION
**Pull request by amirfefer**
`GlobalState` can be a confusing term, `WizardContext` describes better the domain

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ Provisioning frontend service for `cloud.redhat.com`
 ## Development
 This project is based on react and uses the `cloud.redhat.com` micro-frontend architecture
 
-[patternfly](https://www.patternfly.org/v4/) - an opinionated UX/UI library for enterprise applications
-[insights-chrome](https://github.com/RedHatInsights/insights-chrome) - a wrapper that provides an api for the global application (menus / auth / etc)
-[react-query](https://react-query.tanstack.com) - Manages the server state
-[react-tracked](https://react-tracked.js.org) - Optimizes re-renders for a global state management
+[patternfly](https://www.patternfly.org/v4/) - an opinionated UX/UI library for enterprise applications </br>
+[insights-chrome](https://github.com/RedHatInsights/insights-chrome) - a wrapper that provides an api for the global application (menus / auth / etc) </br>
+[react-query](https://react-query.tanstack.com) - Manages the server state </br>
```
Comment (ezr-ondrej) [inline README.md:10]: nitpick: I don't think `</br>` is good use here :thinking:

File: docs/dev.md [hunk 2]
```diff
@@ -22,34 +22,36 @@ First have a look in the `API/index.js` to see if there's already a fetch functi
    ```
 2. Add a specific constant or a function for the request's key in `API/keys.js`, this allow caching data and prevents redundant fetching, for further information about creating a query key, see [queries keys docs](https://react-query.tanstack.com/guides/query-keys)
 3. Consume the hook `useQuery` in your component
-   ```js
+   
+```js
      const {
```
Comment (ezr-ondrej) [inline docs/dev.md:27]: Isn't the indent wrong here? :thinking:

**Review by ezr-ondrej — APPROVED** [review]
Just two nitpicks, but I'm ok to merge with those as well :+1:

**Review by ezr-ondrej — CHANGES_REQUESTED** [review]
Needs rebase and change also usage that was introduced in #26

**Comment by ezr-ondrej** [comment]
@amirfefer can we have a rebase here?

**Comment by adiabramovitch** [comment]
Changes in `DescriptionListAWS` are missing

**Review by adiabramovitch — CHANGES_REQUESTED** [review]
Changes in `DescriptionListAWS` are missing

File: README.md (hunk 1 above)
Comment (amirfefer) [reply README.md:10]: why not?

Comment (ezr-ondrej) [reply README.md:10]: Why not use a Markdown native utils for it, like proper list for example?

Comment (adiabramovitch) [reply README.md:10]: Is there a decision? 😺

Comment (ezr-ondrej) [reply README.md:10]: I still disagree with this, but it's so minor, I'll not block on it :)

**Review by ezr-ondrej — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (ezr-ondrej) [reply README.md:10]: @amirfefer you've said offline you will use the list, shall we wait for that change?

**Review by ezr-ondrej — DISMISSED** [review]
Thanks @amirfefer !

**Comment by ezr-ondrej** [comment]
@adiabramovitch is everything addressed? :)

**Comment by adiabramovitch** [comment]
I am getting:
image
Is it just me? @ezr-ondrej , @amirfefer

**Comment by amirfefer** [comment]
oops, can you please try  again @adiabramovitch  :)

**Review by adiabramovitch — APPROVED** [review]
Thanks! @amirfefer

**Pull request merged**